src/CWL/Decode.fs

```diff
@@ -42,13 +48,24 @@ module Decode =
             outputBinding
         )
 
+    let direntDecoder: (YAMLiciousTypes.YAMLElement -> CWLType) =
+        Decode.object (fun get ->
+            Dirent
+                {
+                    // BUG: Entry Requires an Entryname to be present when it's an expression
```

Comment:
Can you explain this, are you pushing code you know having a bug? 😆

Reply:
I know that it is theoretically possible, but don't know how it would look/work then.